.env.example

```
@@ -564,3 +564,13 @@ ALPHA_INVITE_DECAY_MAX_DECLINES=5
 ALPHA_PER_NPC_CUSTOMIZATION_ENABLED=true
 # Enable grandfathering for existing members when thresholds change
 ALPHA_GRANDFATHERING_ENABLED=true
+
+# ============================================
+# Stripe
+# ============================================
+STRIPE_SECRET_KEY=sk_test_...
+NEXT_PUBLIC_STRIPE_PUBLISHABLE_KEY=pk_test_...
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The NEXT_PUBLIC_STRIPE_PUBLISHABLE_KEY key should go before the STRIPE_SECRET_KEY key

(UnorderedKey)